_data/conf/main.yml

```diff
@@ -92,7 +92,7 @@ search_with_uri: true
 #                Image Viewer
 ###########################################################
 # this enables image viewer
-image_viewer: true
+image_viewer: false
```

Comment:
Would you mind to explain this change?

Reply:
This improves some of the performance to the web page.

Reply:
This comment should be in the code, so anybody can understand it while reading it.